include/onnxruntime/core/providers/winml/winml_provider_factory.h

```diff
@@ -3,7 +3,10 @@
 
 #include "onnxruntime_c_api.h"
```

Comment:
Can we add a comment here clarifying that this is a private header file, not meant to be called directly? (otherwise it's weird that the DML header in the same folder is publicly callable but this one doesn't indicate so anywhere)

Reply:
Sure, this is a strange artifact of how onnxruntime exports methods. In order for the winml adapter to show up it needs to pretend it is an execution provider.

Dml is in its own dml ep folder, so i dont think people will get confused in practice, and this header is not packages in a nuget.

Reply:
Why do you call this a private header file?

Reply:
@yuslepukhin : Private in the sense that general applications are not supposed to call it directly, but used through WinML.